lib/generators/solidus_paypal_commerce_platform/install/install_generator.rb

```diff
@@ -26,11 +27,13 @@ def mount_engine
       end
 
       def run_migrations
-        run_migrations = options[:auto_run_migrations] || ['', 'y', 'Y'].include?(ask('Would you like to run the migrations now? [Y/n]')) # rubocop:disable Layout/LineLength
-        if run_migrations
-          run 'bin/rails db:migrate'
-        else
-          puts 'Skipping bin/rails db:migrate, don\'t forget to run it!' # rubocop:disable Rails/Output
+        unless options[:skip_migrations]
```

Comment:
what about 

```
return if options[:ship_migrations]
```

We'd avoid indenting the whole method this way and I think it's more readable.

Reply:
Yep, totally agree! Made the change.